template/profile.html

```diff
@@ -0,0 +1,130 @@
+<dom-module id="user-game">
+  <template>
+    <mdl-card class="user-game" shadow="4">
+      <img src="" />
+      <mdl-card-title-text class="user-game-name"></mdl-card-title-text>
+    </mdl-card>
+  </template>
+  <style>
+    .user-game.mdl-card {
+      width: 200px;
+      height: 200px;
+      display: inline-block;
+      margin-left: 10px;
+      margin-right: 10px;
+      transition: all .2s ease-in-out;
+    }
+
+    .user-game.mdl-card:hover { transform: scale(1.1); }
+
+    .user-game img {
+      width: 200px;
+      height: 150px;
+      background-color: lightgray;
+    }
+
+    .user-game-name{
+      margin-left: 10px;
+      margin-top: 10px;
+    }
+  </style>
+
+  <script>
+    Polymer({
+      is: "user-game",
+    })
+  </script>
+</dom-module>
+
+<dom-module id="user-achievement">
+  <template>
+    <mdl-chip class="user-achievement">
+      <mdl-chip-contact class="mdl-color--amber mdl-color-text--white user-achievement-image"></mdl-chip-contact>
+      <mdl-chip-text class="user-achievement-name"></mdl-chip-text>
+    </mdl-chip>
+  </template>
+  <style>
+  </style>
+
+  <script>
+    Polymer({
+      is: "user-achievement",
+    })
+  </script>
+</dom-module>
+
+<dom-module id="papan-user-profile" class="user-page">
+  <template>
+    <mdl-card class="user-info">
+      <img src="" id="picture"/>
+      <mdl-card-title-tex id="nickname">
+        Nickname
+      </mdl-card-title-tex>
+    </mdl-card>
+
+    <div id="user-games-list">
+      <div class="category">Games</div>
+    </div>
+
+    <div id="user-achievements-list">
+      <div class = "category">Achievements</div>
+    </div>
+  </template>
+
+  <style>
+    .user-info {
+      width: auto;
+      height: 250px;
+    }
+
+    .user-info #nickname{
+      margin-top: 10px;
+      margin-left: 10px;
+      font-size: 20pt;
+    }
+
+    .user-info img {
+      margin-top: 10px;
+      margin-left: 10px;
+      width: 200px;
+      height: 200px;
+      background-color: lightgray;
+    }
+
+    .category {
+      font-size: 20pt;
+      border-bottom: 1px solid lightgrey;
+      margin-top: 10px;
+      margin-bottom: 10px;
+      padding-bottom: 5px;
+    }
+  </style>
+
+  <script>
+    Polymer({
+      is: "papan-user-profile",
+
+      ready: function() {
+        //populate list of games
+        var list = document.getElementById('user-games-list');
+        var games = ["Chess", "Tic tac toe", "Falken's Maze", "Poker"];
+        for (i = 0; i < games.length; i++) {
```

Comment:
Using dom-repeat into the template (or a component which can render lists) will spare you the loops.